warehouse/templates/packaging/detail.html

```diff
@@ -90,6 +90,7 @@ <h1 class="package-header__name">{{ release.project.name }} {{ release.version }
   </div>
 </section>
 
+{% if release.urls.values()|filter_valid_uris|length > 0 %}
```

Comment:
It seems like you want to print this if there are *any* valid URLs, right? So why not instead write the filter to return a boolean, e.g.,

```py
def any_valid_uris(uris):
     return any(is_valid_uri(uri) for uri in uris)
```

And use it like:

```jinja2
{% if release.urls.values() |any_valid_urls %}
...
{% endif %}
```